### ISSUE TITLE
--ignore-redirects not functioning properly

### ISSUE THREAD

**Issue by WendallHersh** [carlowahlstedt/NewmanPostman_VSTS_Task#54]
I have the --ignore-redirects flag set in the Newman task. "--ignore-redirects" is being sent in the command 

[command]C:\windows\system32\cmd.exe /D /S /C "C:\npm\prefix\newman.cmd run "D:\a\r1\a\Staging Artifacts\automation\postman\collections\RegressionTestsSPI.postman_collection" -r "cli,junit" -n 1 --globals "D:\a\r1\a\Staging Artifacts\automation\postman\globals\globals.postman_globals" --ignore-redirects -e "D:\a\r1\a\Staging Artifacts\automation\postman\environments\staging.postman_environment""

The redirects are still happening in the tests though. The tests work fine in Postman and Newman locally.

**Comment by carlowahlstedt** [comment]
Would you mind to post the same run command you've copied from Azure Pipelines but the version that you've got working locally?

**Comment by WendallHersh** [comment]
I think it might have been a quirk of Azure at the time that I posted this. It is now currently working, although I have stopped using the Newman CLI integration in preference of straight "Command Line".

But here is the copied Newman run statement that I run locally. 

`newman run RegressionTestsSPI.postman_collection -e environments/staging.postman_environment -g globals/globals.postman_globals --ignore-redirects -r cli,junit
`
